### PR TITLE
Fixes AR demo, makes it a VR AR demo

### PR DESCRIPTION
**Pull request by Kuranes**

### CODE REVIEW

**Comment by pierreant-p** [comment]
gg. waiting for update of the api to merge

File: ar/main-0.0.1.js [hunk 1]
```diff
@@ -132,75 +144,95 @@ stopButton.addEventListener('click', function() {
...
+                clientApi.setMatrix(rootNode, m);
+
+
+            }
         }
 
 
     }, 15);
-};
+};
```
Comment (cedricpinson) [inline ar/main-0.0.1.js]: configure your editor to keep end of line. it avoids useless change line

**Comment by cedricpinson** [comment]
I think it's merged @Kuranes you can check if the code is in master

**Comment by Kuranes** [comment]
I confirm: new API code is in master

**Pull request merged**